src/components/app/content/index.js

```diff
@@ -5,25 +5,21 @@ import Cards from "./cards";
 function Content() {
     const [readOnly, setReadOnly] = useState(false)
     const [cards, setCards] = useState([
-        {id: 1, header: 'Caption', body: 'Text...', checked: false, editable: false},
-        {id: 2, header: 'Caption', body: 'Text...', checked: false, editable: false},
-        {id: 3, header: 'Caption', body: 'Text...', checked: false, editable: false},
-        {id: 4, header: 'Caption', body: 'Text...', checked: false, editable: false},
-        {id: 5, header: 'Caption', body: 'Text...', checked: false, editable: false},
-        {id: 6, header: 'Caption', body: 'Text...', checked: false, editable: false},
-        {id: 7, header: 'Caption', body: 'Text...', checked: false, editable: false},
-        {id: 8, header: 'Caption', body: 'Text...', checked: false, editable: false},
```

Comment:
Well, `checked` property could have been stored in Content state for future tasks... but let's see where would this lead to.